<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PleinDeFormesClass</class>
 <widget class="QMainWindow" name="PleinDeFormesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STS La Providence</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="MainLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="TitleLabel">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Ajouter une personne</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="FormContainerLayout">
        <item>
         <layout class="QFormLayout" name="FormLayoutLeft">
          <item row="0" column="0">
           <widget class="QLabel" name="NomLabel">
            <property name="text">
             <string>Nom :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="NameEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="FirstNameLabel">
            <property name="text">
             <string>Prénom :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="FirstNameEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="AdressLabel">
            <property name="text">
             <string>Adresse :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="AdresseEdit"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="AdresseEdit2"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="AdresseEdit3"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="ProfButton">
              <property name="text">
               <string>Professeur</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="EtudButton">
              <property name="text">
               <string>Etudiant</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="TitreLabel">
            <property name="text">
             <string>Titre :</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="FormLayoutRight">
          <item row="0" column="0">
           <widget class="QLabel" name="AgeLabel">
            <property name="text">
             <string>Age :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="AgeEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="PhoneLabel">
            <property name="text">
             <string>Téléphone :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="PhoneEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="EmailLabel">
            <property name="text">
             <string>Email :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="MailEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="BaseHumainForm" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okBtn">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAjouter">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionAjouter"/>
    <addaction name="actionAffichage"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuAjouter"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAjouter">
   <property name="text">
    <string>Ajouter</string>
   </property>
  </action>
  <action name="actionAffichage">
   <property name="text">
    <string>Affichage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseHumainForm</class>
   <extends>QWidget</extends>
   <header>basehumainform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PleinDeFormes.qrc"/>
 </resources>
 <connections/>
</ui>
